src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index];
     }
```

Comment:
Consider storing the result of `Color.values()` in a local variable before using it in `nextInt` to avoid calling `values()` method multiple times which creates a new array each time it's called. This can be a performance issue as it unnecessarily creates multiple objects.